app/views/access_limit/edit.html.erb

```diff
@@ -2,6 +2,25 @@
 <% content_for :browser_title, t("access_limit.edit.browser_title") %>
 <% content_for :back_link, render_back_link(href: document_path(@edition.document)) %>
 
+<% service = LinkablesService.new("organisation") %>
+
+<% if @edition.primary_publishing_organisation_id %>
+  <% primary_org_id = @edition.primary_publishing_organisation_id %>
+  <% primary_org = service.by_content_id(primary_org_id).fetch("internal_name") %>
```

Comment:
This will still error on a nil, you'll need to use the safe navigation operator before the fetch

Reply:
Oops - going to quickly. Fixed now.